examples/Example/src/bottomSheet/index.tsx

```diff
@@ -50,51 +50,51 @@ export class BottomSheet extends Component<{}, StateType> {
       lastSnap: END,
     };
 
-    this._lastScrollYValue = 0;
-    this._lastScrollY = new Animated.Value(0);
-    this._onRegisterLastScroll = Animated.event(
-      [{ nativeEvent: { contentOffset: { y: this._lastScrollY } } }],
+    this.lastScrollYValue = 0;
+    this.lastScrollY = new Animated.Value(0);
+    this.onRegisterLastScroll = Animated.event(
+      [{ nativeEvent: { contentOffset: { y: this.lastScrollY } } }],
       { useNativeDriver: USE_NATIVE_DRIVER }
     );
-    this._lastScrollY.addListener(({ value }) => {
-      this._lastScrollYValue = value;
+    this.lastScrollY.addListener(({ value }) => {
+      this.lastScrollYValue = value;
     });
 
-    this._dragY = new Animated.Value(0);
-    this._onGestureEvent = Animated.event(
-      [{ nativeEvent: { translationY: this._dragY } }],
+    this.dragY = new Animated.Value(0);
+    this.onGestureEvent = Animated.event(
+      [{ nativeEvent: { translationY: this.dragY } }],
       { useNativeDriver: USE_NATIVE_DRIVER }
     );
 
-    this._reverseLastScrollY = Animated.multiply(
+    this.reverseLastScrollY = Animated.multiply(
       new Animated.Value(-1),
-      this._lastScrollY
+      this.lastScrollY
     );
 
-    this._translateYOffset = new Animated.Value(END);
-    this._translateY = Animated.add(
-      this._translateYOffset,
-      Animated.add(this._dragY, this._reverseLastScrollY)
+    this.translateYOffset = new Animated.Value(END);
+    this.translateY = Animated.add(
+      this.translateYOffset,
+      Animated.add(this.dragY, this.reverseLastScrollY)
     ).interpolate({
       inputRange: [START, END],
       outputRange: [START, END],
       extrapolate: 'clamp',
     });
   }
-  _onHeaderHandlerStateChange = ({
+  onHeaderHandlerStateChange = ({
```

Comment:
Here and in other places you remove `_` from functions, and they are not marked as private explicitly. Wouldn't this change mean that now this function is a public method, which, I believe, is not true? cc @jakub-gonet @jkadamczyk

Reply:
It is public anyway just the name is different. What will be best I guess is marking it private with TS private modifier